docs/src/main/sphinx/release/release-360.md

```diff
@@ -0,0 +1,71 @@
+# Release 360 (XXX Jul 2021)
+
+## General
+
+* Reduce number of requests from coordinator to workers ({issue}`8460`)
+* Improve query performance by sending collected dynamic filters from coordinator to workers. ({issue}`5183`)
+* Improve performance of inequality joins where join condition sides have different type. ({issue}`8500`)
+* Fix query failure when there is a constant predicate on some
+  column `col` (e.g `col=1`), followed by  `ORDER BY col` and `LIMIT`. ({issue}`8535`)
+* Fix SHOW CREATE SCHEMA. ({issue}`8598`)
+* Fix query failure when running `SHOW CREATE SCHEMA information_schema`. ({issue}`8600`)
+* Improve performance of queries with `WHERE` predicates. ({issue}`8624`)
+* Disallow `DROP SCHEMA` if the schema is not empty. ({issue}`8660`)
+  
+## Security
+
+* Add support for automatic configuration of TLS for {doc}`/security/internal-communication`. This removes the need to provision
+  per-worker TLS certificates. ({issue}`7954`)
+  
+## CLI
+
+* Fix auto completion when pressing the tab button. ({issue}`8529`)
+
+## Hive connector
+
+* Ensure generation of unique ROW IDs during UPDATE on ORC ACID transactional tables.
+  Before it was possible that Hive was not able to read the table updated by Trino ({issue}`8268`)
```

Comment:
That doesn't describe the fix from the end user perspective.

I think it's fair to say that we were violating Hive table format "spec" (where spec = Hive implementation), so it would be fair to say "Fix data corruption when performing UPDATE on ORC ACID transactional table".